patches/chrome-browser-ui-views-location_bar-selected_keyword_view.cc.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/chrome/browser/ui/views/location_bar/selected_keyword_view.cc b/chrome/browser/ui/views/location_bar/selected_keyword_view.cc
+index f824957011655e33004ff528d5a791878cbb3868..77ce646a829661087760894119495f4d198130c0 100644
+--- a/chrome/browser/ui/views/location_bar/selected_keyword_view.cc
++++ b/chrome/browser/ui/views/location_bar/selected_keyword_view.cc
+@@ -130,6 +130,7 @@ void SelectedKeywordView::SetCustomImage(const gfx::Image& image) {
+     vector_icon = &vector_icons::kGoogleAgentspaceMonochromeLogoIcon;
+ #endif
+   }
++  BRAVE_SELECTED_KEYWORD_VIEW_SET_CUSTOM_IMAGE
```

Comment:
`#define kSearchAggregator kSearchAggregator) { /* this is guarded by GOOGLE_CHROME_BRANDING anyway */ } if (...) { ... } else if (false`

Reply:
Would it be OK to hang this off of `kAiMode` (as in current version), since it's a bit more closely related?